sdk/core/azure-core/test/ut/context.cpp

```diff
@@ -18,7 +18,8 @@ TEST(Context, Basic)
   Context context;
   Context::Key const key;
 
-  EXPECT_FALSE(context.HasKey(key));
+  int placeholder;
+  EXPECT_FALSE(context.TryGetValue<int>(key, placeholder));
```

Comment:
Btw, it is possible that we can write just `TryGetValue(...)` or `TryGetValue<>(...)`, and a compiler will deduce the value of `<T>`.

Reply:
Verify that the value hasn't changed when `false` is returned.

Reply:
> Btw, it is possible that we can write just `TryGetValue(...)` or `TryGetValue<>(...)`, and a compiler will deduce the value of `<T>`.

Verified that's possible. Good call. I will leave it as explicit in tests though.